common/crypto/attestation-api/README.md

```diff
@@ -0,0 +1,91 @@
+# Attestation API
+
+This library implements a single API for generating and verifying hardware-based attestation (e.g., Intel SGX EPID/DCAP attestations).
+The library implementation originally started as part of the Fabric Private Chaincode project, and later spun off and extended.
+
+The library works with Intel SGX EPID attestations, verified through Intel Attestation Service (IAS), and DCAP attestations, verified through Intel Trust Authority (ITA).
+
+## Build
```

Comment:
In my view, this should be still be a standalone component, so that if you want to just build it, then... set up the environment and run the cmake build. It should still work

Reply:
Hmm, that does not contradict necessarily what i said? This readme should say this and the top-level README should reference this component's integration?  Certainly as you seem to rely on parts of the README also for the artifact provisioning also necessary at the top-level?